packages/substrate-bindings/src/codecs/metadata/normalized.ts

```diff
@@ -0,0 +1,161 @@
+import { CodecType } from "scale-ts"
+import { V14Lookup } from "./lookup"
+import { storageMap } from "./pallets"
+import { HexString } from "../scale"
+import { itemDeprecation, variantDeprecation } from "./deprecation"
+import { viewFunction } from "./runtime-api"
+import { V14 } from "./v14"
+import { V16 } from "./v16"
+import { V15 } from "./v15"
+import { Metadata } from "./metadata"
+
+type Entrypoint<T> =
```

Comment:
I was about to suggest using `DeprecationInfo` for `Entrypoint`, but I saw that the inner codec is different.

I'd suggest a different name then, to show that this is the... Entrypoint(?) but only for Enum/Variant types (calls/events/errors). Not sure if EnumEntrypoint is the best either... Maybe `EnumReference`?